navigation/main.json

```diff
@@ -83,6 +83,18 @@
           "alt": "Proof of concept"
         },
         "type": "icon"
+      },
+      {
+        "name": "Ecommerce Development Services",
```

Comment:
Remove 'Services' from here. This is a category 'Services' so you don’t need to show again